src/main/java/com/kodilla/ecommercee/OrderController.java

```diff
@@ -22,37 +20,82 @@
     public OrderController() {
     }
 
+    private List<ProductDto> initializeSampleProducts() {
+        List<ProductDto> products = new ArrayList<>();
+        String[] names = {"Ubrania", "Kostmetyki", "Samochdy", "Elektronika"};
+        String descripiton = "Dummy item description.";
+        for (int id = 0; id < names.length; id++) {
+            products.add(new ProductDto(
+                    (long) (id + 1),
+                    names[id],
+                    descripiton,
+                    new BigDecimal(id*100),
+                    (long)id
+            ));
+        }
+        return products;
+    }
+
+    private List<OrderDto> generateOrders() {
+        List<OrderDto> orders = new ArrayList<>();
+        LocalDateTime purchaseDate = LocalDateTime.of(2019, Month.SEPTEMBER, 1, 10, 30, 0);
+        LocalDateTime deliveryDate = purchaseDate.plusDays(5);
+        List<ProductDto> products = initializeSampleProducts();
+        for (int i = 0; i < products.size(); i++) {
+            List<ProductDto> productsList = new ArrayList<>();
+            Long orderId = Long.valueOf(i + 1);
+            for (int j = 0; j <= i; j++) {
+                productsList.add(products.get(j));
+
+            }
+            orders.add(new OrderDto(orderId, 1L, 2L, purchaseDate, productsList, "DELIVERED", deliveryDate, orderValue(productsList)));
+        }
+        return orders;
+    }
+
+    private BigDecimal orderValue(List<ProductDto> products) {
+        BigDecimal output = new BigDecimal(0);
+
+        for (ProductDto product : products) {
+            output.add(product.getPrice());
```

Comment:
ta pętla nie sumuje wartości produktów; aby to robić, musiałaby wyglądać następująco:
`output = output.add(product.getPrice());`  
najprostszym rozwiązaniem jednak wydaje się:
`return products.stream().map(ProductDto::getPrice).reduce(BigDecimal.ZERO, BigDecimal::add);`